templates/zerver/base.html

```diff
@@ -14,8 +14,8 @@
                 {% endif %}
             {% endif %}
         {% endblock %}
-        <link id="favicon" rel="icon" href="/static/images/favicon.svg?v=4" />
-        <link rel="alternate icon" href="/static/images/favicon.png?v=4" />
+        <link id="favicon" rel="icon" href="{{ static('images/favicon.svg?v=4') }}" />
+        <link rel="alternate icon" href="{{ static('images/favicon.png?v=4') }}" />
```

Comment:
This URL-encodes the `?v=4` as if it’s part of the filename, resulting in 404s.

```suggestion
        <link id="favicon" rel="icon" href="{{ static('images/favicon.svg') }}?v=4" />
        <link rel="alternate icon" href="{{ static('images/favicon.png') }}?v=4" />
```